src/apack/zif_abapgit_apack_definitions.intf.xml

```diff
@@ -4,7 +4,6 @@
   <asx:values>
    <VSEOINTERF>
     <CLSNAME>ZIF_ABAPGIT_APACK_DEFINITIONS</CLSNAME>
-    <VERSION>1</VERSION>
     <LANGU>E</LANGU>
     <DESCRIPT>APACK: Type Definitions</DESCRIPT>
     <EXPOSURE>2</EXPOSURE>
```

Comment:
Can we git rid of EXPOSURE in INTF too?

Reply:
Since all interfaces are supposed to be PUBLIC (EXPOSURE=2), I agree that EXPOSURE can be removed from the XML. The used API is using 2 by default and the given value is ignored. Hopefully this is not changed in future. Maybe we should pass 2 to the API when the interface is to be de-serialized.

Reply:
Maybe we should so with UNICODE as well. Are there any classes/interfaces/reports alive that do not have 'X' as unicode flag? Idea: Remove it from the XML and pass 'X' to the APIs for de-serialization. Any concerns?

Reply:
I like that idea!

We can do the same with STATE of VSEOCLAS and VSEOINTERF, can't we? It will always be 1, right?

Reply:
I discussed the UNICODE and STATE flag with the CLAS/INTF responsible at SAP:
For STATE, '1' can be used as fix value. '2' should never be used and '0' is out-dated since years. So we can/should go for '1'!
For UNICODE there is no recommendation to use 'X' in all cases. There might be customer objects without unicode flag. The question is: If 'X' is used all the time, the object in the import system is different to the object in the source source. If this is okay, 'X' can be used. But if the customer expects to have '0' in the destination system, 'X' is wrong. It depends on the abapGit community to decide how abapGit should behave: Use 'X' all the time or differentiate between '0' and '1'